socialdistribution/socialdistribution/settings.py

```diff
@@ -0,0 +1,127 @@
+"""
+Django settings for socialdistribution project.
+
+Generated by 'django-admin startproject' using Django 4.1.6.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-$_36yrktyy9k)66uqv1kx9gx7h2)2a^du_mp$55pjd1w(17hbg'
```

Comment:
Are we ok with leaving this in? Usually, we put this in a secrets.py folder which goes in the gitignore. Obviously, it doesn't matter too too much for this but it's worth mentioning.

Reply:
Should be okay for now, we can do this later. Please review and merge.